packages/pwa-kit-dev/package-lock.json

```diff
@@ -2389,14 +2389,232 @@
 			}
 		},
 		"array.prototype.flatmap": {
-			"version": "1.3.0",
```

Comment:
Thanks for your contribution @breadadams
The change makes sense, but looking through the code I see thousands of lines changed in the package-lock.json file that are unrelated to the change.

This is most likely because you used `npm` to bump the `eslint-plugin-react` version.

We use `lerna` to manage dependencies in the monorepo and minimize the number of unintended changes in the `package-lock.json` files.

That way we can target updating lock files intentionally in specific PRs.

Do you mind re-creating the change using `lerna` to bump the `eslint-plugin-react` version and see if that results in fewer changes in the `package-lock.json` file?

You can run this `lerna` command at the root of the monorepo:

```
npm run lerna -- add eslint-plugin-react@7.32.2 packages/pwa-kit-dev/
```

Reply:
Thanks @adamraya, yep my bad I installed with `npm`. Much cleaner diff now 😉

Reply:
@adamraya I've just tried updating the dependency with [`lerna-update-wizard`](https://www.npmjs.com/package/lerna-update-wizard) and the huge diffs are back. Any suggestions? Thanks

Reply:
@breadadams I reproduced the same changes in the lock files locally, so that seems to be the minimal set of changes required for the bump.

Good catch bumping the version on both `pwa-kit-dev` and `internal-lib-build`.

I tested things locally, and things look good to go. 
I'll merge the PR into `develop` and it'll be included in the next minor release v2.7.0.

Thanks again for your help here.